packages/ubuntu_wizard/pubspec.yaml

```diff
@@ -11,13 +11,16 @@ dependencies:
   collection: ^1.15.0
   crypt: ^4.0.1
   diacritic: ^0.1.3
+  file: ^6.1.2
   flutter:
     sdk: flutter
   flutter_localizations:
     sdk: flutter
   form_field_validator: ^1.1.0
   intl: ^0.17.0
   password_strength: ^0.2.0
+  path: ^1.8.0
+  plugin_platform_interface: ^2.1.2
```

Comment:
only used for testing url launcher -> move to `dev_dependencies`

Reply:
✅

Reply:
did you forget to move this?

Reply:
(`plugin_platform_interface` is only used in `packages/ubuntu_wizard/test/url_launcher_test.dart`)

Reply:
this is still unresolved